openjdk/excludes/ProblemList_openjdk18-openj9.txt

```diff
@@ -0,0 +1,46 @@
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+############################################################################
+
+# To determine required OS exclusion string see http://hg.openjdk.java.net/code-tools/jtreg/file/6f00c63c0a98/src/share/classes/com/sun/javatest/regtest/config/OS.java
+
+############################################################################
+
+# jdk_foreign
+
+java/foreign/SafeFunctionAccessTest.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestDowncall.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestIntrinsics.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestNULLAddress.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestNative.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/malloc/TestMixedMallocFree.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/virtual/TestVirtualCalls.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+
+java/foreign/StdLibTest.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestUpcall.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestUpcallException.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestUpcallHighArity.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/TestUpcallStructScope.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+
+java/foreign/TestVarArgs.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/valist/VaListTest.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+
+java/foreign/TestIllegalLink.java https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/stackwalk/TestAsyncStackWalk.java#default_gc https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/stackwalk/TestAsyncStackWalk.java#shenandoah https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/stackwalk/TestAsyncStackWalk.java#zgc https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/stackwalk/TestStackWalk.java#default_gc https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/stackwalk/TestStackWalk.java#shenandoah https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
+java/foreign/stackwalk/TestStackWalk.java#zgc https://github.com/eclipse-openj9/openj9/issues/13946 generic-all
```

Comment:
re https://github.com/eclipse-openj9/openj9/issues/13946#issuecomment-971927455: Only the `*StackWalk` tests should be excluded since they are unsupported. Other tests pass once the assertion is removed so they should not be excluded.

https://github.com/eclipse-openj9/openj9/issues/13946 covers a number of different failures. Instead of https://github.com/eclipse-openj9/openj9/issues/13946, https://github.com/eclipse-openj9/openj9/issues/13993 should be used as the associated issue for excluding `*StackWalk` failures.

Reply:
Re-enabling tests will be difficult if we link all test exclusions to https://github.com/eclipse-openj9/openj9/issues/13946. So, a Github issue should be opened for each unique failure before excluding associated tests.

Reply:
Agreed and associated [#13993](https://github.com/eclipse-openj9/openj9/issues/13993) with *StackWalk failures.
Will create a couple of issues for other cases to be excluded.

Reply:
> Will create a couple of issues for other cases to be excluded.

Already creating Github issues for MH related failures. I will tag you once they are created.